Dockerfile

```diff
@@ -1,12 +1,14 @@
-FROM alpine:3.5
+FROM vertigo/go-builder as builder
 
 ADD . /go-ethereum
 RUN \
-  apk add --update git go make gcc musl-dev linux-headers && \
   (cd go-ethereum && make geth)                           && \
   cp go-ethereum/build/bin/geth /geth                     && \
```

Comment:
There's no reason to copy geth to a different location, you can copy it from it's build path directly into the final image.